### PR TITLE
Add GitHub documentation and templates.

### PR DESCRIPTION
**Pull request by pento**
## Description

This PR adds the following documentation, broadly based on the equivalent docs from Gutenberg:
- CONTRIBUTING.md
- Three issue templates: Bug Report, Feature Request, and Question
- A pull request template

It also moves the release process documentation out of README.md, into its own file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Getting Started
+
+TestPress is an Electron application, built primarily in JavaScript.
+
+To get started, you will need to [Node.js installed](https://nodejs.org/en/). Most MacOS and Linux systems will have Node already installed, but you may need to download and install it on Windows.
+
+TestPress can sometimes need a different version of Node than you have on your system, you can use [nvm](https://github.com/creationix/nvm) to change node versions on the command line:
+
+```
+npx nvm install
```
Comment (talldan) [inline CONTRIBUTING.md]: Tested locally, and running nvm via npx didn't work.

```suggestion
nvm install
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Thank you for thinking about contributing to TestPress! If you're unsure of anything, know that you're 💯 welcome to submit an issue or pull request on any topic. The worst that can happen is that you'll be politely directed to the best location to ask your question or to change something in your pull request. We appreciate any sort of contribution and don't want a wall of rules to get in the way of that.
+
+As with all WordPress projects, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+All WordPress projects are [licensed under the GPLv2+](/LICENSE.md), and all contributions to TestPress will be released under the GPLv2+ license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv2+ license.
+
+This document covers the technical details around setup, and submitting your contribution to the TestPress project.
+
+## Getting Started
```
Comment (talldan) [inline CONTRIBUTING.md]: It may also be worth mentioning that docker is needed as well.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pento) [reply CONTRIBUTING.md]: Huh, interesting. We should probably fix the Gutenberg docs, too. 🙂

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+To get started, you will need to [Node.js installed](https://nodejs.org/en/). Most MacOS and Linux systems will have Node already installed, but you may need to download and install it on Windows.
+
+TestPress can sometimes need a different version of Node than you have on your system, you can use [nvm](https://github.com/creationix/nvm) to change node versions on the command line:
+
+```
+npx nvm install
+```
+
+You also should have the latest release of [npm installed][npm]. npm is a separate project from Node.js and is updated frequently. If you've just installed Node.js which includes a version of npm within the installation you most likely will need also to update your npm installation. To update npm, type this into your terminal: `npm install npm@latest -g`
+
```
[the remark this answers is not in the document]
Comment (pento) [reply CONTRIBUTING.md]: That's a much nicer command to read than the other one. 👍🏻

File: CONTRIBUTING.md (hunk 2 above)
Comment (pento) [reply CONTRIBUTING.md]: 👍🏻

**Pull request merged**